boards/DCM/Src/App/states/App_FaultState.c

```diff
@@ -4,6 +4,53 @@
 
 #include "App_SharedMacros.h"
 
+#define CLEAR_INV_FAULT_PARAM_ADDRESS (20U)
+#define WRITE_CMD (1U)
+#define CLEAR_INV_FAULT_MSG()                                          \
+    {                                                                  \
+        .d1_parameter_address_command = CLEAR_INV_FAULT_PARAM_ADDRESS, \
+        .d2_read_write_command = WRITE_CMD, .d3_data_command = 0       \
+    }
+
+/**
+ * Send the clear fault command to an inverter if requested by a PCAN node
+ * @param can_tx CAN TX interface
+ * @param can_rx CAN RX interface
+ * @note Should only be called when the inverter is disabled (see pg. 38 in the
+ * Software UM)
+ */
+static void App_HandleClearInvFaultsCmd(
```

Comment:
since this is only used in the fault state we prob do not need this fxn anymore, can just implement in fault state directly

Reply:
> offline discussion: if no concerns, can power inv 12V from run on entry since we no longer run into faults when powering LV b4 HV. @joethrstn @Eddy13013 @Auhv @timbitss do u see any potential issues w this

I don't see a concern since the DCM only enter the DRIVE state and transmit non-zero torque requests once the BMS is in the DRIVE state

Reply:
done